### PR TITLE
hURL #3007

### PR DESCRIPTION
**Pull request by dualfade**
please review --

### CODE REVIEW

File: packages/hURL/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=hURL
```
Comment (Edu4rdSHL) [inline packages/hURL/PKGBUILD:4]: Please always put `pkgname` in lowercase.

File: packages/hURL/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+pkgver=20.afca9c5
+pkgrel=1
+pkgdesc='Hexadecimal & URL (en/de)coder.'
+arch=('any')
+groups=('blackarch' 'blackarch-misc' 'blackarch-exploitation')
+url='https://github.com/fnord0/hURL'
+license=('custom:unknown')
+depends=('perl' 'perl-cgi' 'perl-uri' 'perl-digest-sha1' 'perl-mime-types')
+makedepends=('git')
+source=("git+https://github.com/fnord0/$pkgname.git")
```
Comment (Edu4rdSHL) [inline packages/hURL/PKGBUILD:14]: `source=("$pkgname::git+https://github.com/fnord0/hURL.git")`

File: packages/hURL/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+build() {
+  cd $pkgname
+}
+
+package() {
+  cd $pkgname
+
+  install -Dm 755 "$srcdir/$pkgname/$pkgname" "$pkgdir/usr/bin/$pkgname"
```
Comment (Edu4rdSHL) [inline packages/hURL/PKGBUILD:30]: `install -Dm 755 fURL "$pkgdir/usr/bin/$pkgname"`

File: lists/to-release [hunk 4]
```diff
@@ -0,0 +1 @@
+hURL
```
Comment (Edu4rdSHL) [inline lists/to-release:1]: `hurl`

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Hey @dualfade, thank you a lot for the PR, please follow the recommendations.

File: packages/hURL/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/hURL/PKGBUILD:4]: @Edu4rdSHL Someone else wrote a new app named hurl ( lower ). I can rename to something else if you prefer ?

Comment (Edu4rdSHL) [reply packages/hURL/PKGBUILD:4]: I didn't saw it in our repos, how's it conflicting?

Comment (dualfade) [reply packages/hURL/PKGBUILD:4]: aur/hurl-bin 1.1.0-1 (+0 0.00) (Installed)
    Hurl, run and test HTTP requests

Binary name is "hurl"

Comment (dualfade) [reply packages/hURL/PKGBUILD:4]: how about hurl-encoder ?

Comment (Edu4rdSHL) [reply packages/hURL/PKGBUILD:4]: Well, that's in AUR, we don't care about names in AUR packages, see https://github.com/BlackArch/blackarch/issues/2992 for conext.

Comment (dualfade) [reply packages/hURL/PKGBUILD:4]: Huh thats fine. Will pork me hah. I'll adjust.

File: packages/hURL/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/hURL/PKGBUILD:14]: adjusted / not committed yet

File: packages/hURL/PKGBUILD (hunk 3 above)
Comment (dualfade) [reply packages/hURL/PKGBUILD:30]: adjusted / not committed yet

File: packages/hURL/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/hURL/PKGBUILD:4]: adjusted / not committed yet

File: packages/hURL/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+prepare() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd $pkgname
+}
+
```
Comment (Edu4rdSHL) [inline packages/hURL/PKGBUILD:26]: Please remove it.

File: packages/hURL/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+arch=('any')
+groups=('blackarch' 'blackarch-misc' 'blackarch-exploitation')
+url='https://github.com/fnord0/hURL'
+license=('custom:unknown')
+depends=('perl' 'perl-cgi' 'perl-uri' 'perl-digest-sha1' 'perl-mime-types')
+makedepends=('git')
+source=("$pkgname::git+https://github.com/fnord0/hURL.git")
+sha512sums=('SKIP')
+
+prepare() {
```
Comment (Edu4rdSHL) [inline packages/hURL/PKGBUILD:17]: The correct name for this function `pkgver()`

Comment (dualfade) [reply packages/hURL/PKGBUILD]: yes my bad. Copy from pkgbuilds --

File: lists/to-release (hunk 4 above)
Comment (Edu4rdSHL) [reply lists/to-release:1]: Please adjust it.

**Review by Edu4rdSHL — APPROVED** [review]
Thank you @dualfade !

**Pull request merged**